configs/components/rubygem-minitar.rb

```diff
@@ -1,11 +1,11 @@
 component "rubygem-minitar" do |pkg, settings, platform|
   # Projects may define a :rubygem_minitar_version setting, or we use 0.6.1 by default:
-  version = settings[:rubygem_minitar_version] || '0.6.1'
+  version = settings[:rubygem_minitar_version] || '0.9'
   pkg.version version
 
   case version
-  when "0.6.1"
-    pkg.md5sum "ce4ee63a94e80fb4e3e66b54b995beaa"
+  when "0.9"
```

Comment:
Bolt and bolt server added the version logic to get the newest minitar. If shared_agent component is on latest we can just get rid of the logic and have a single version. That would involve just deleting the lines in https://github.com/puppetlabs/puppet-runtime/blob/0c65b6f1cc6e178ed607f900dfde67402f4c7a65/configs/projects/bolt-runtime.rb#L11 and https://github.com/puppetlabs/puppet-runtime/blob/0c65b6f1cc6e178ed607f900dfde67402f4c7a65/configs/projects/_shared-pe-bolt-server.rb#L45 and you could change this to something like
```
component 'rubygem-minitar' do |pkg, settings, platform|
  pkg.version '0.9'
  pkg.md5sum '4ab2c278183c9a83f3ad97066c381d84'

  instance_eval File.read('configs/components/_base-rubygem.rb')
end
```